Added Authorization support for someip2val

int0x27 opened the pull request:
- Added target actuator subscriber using kuksa GRPC API (old streaming api)
- cert/ dir contains copies of kuksa.val/cert tools / keys + someip2val specific token.
- JWT token authorization is implemented in CollectorClient
- Added --help argument for someip_feeder showing options and used environment variables.
- Added common simple_log for broker_feeder classes.
- Added env WIPER_STATUS to mute ###|WiperEvent| status lines (or make them print on the same line)
- Fixed vsomeip tag to 3.1.20.3 as current master is not supported

int0x27 committed:
Added Authorization support for someip2val

- Added target actuator subscriber using kuksa GRPC API (old streaming api
- cert/ dir contains copies of kuksa.val/cert tools / keys + someip2val specific token.
- JWT token authorization is implemented in CollectorClient
- Added --help argument for someip_feeder showing options and used environment variables.
- Added common simple_log for broker_feeder classes.
- Added env WIPER_STATUS to mute ###|WiperEvent| status lines (or make them print on the same line)

Signed-off-by: Ivan Kirchev <Ivan.Kirchev@bosch.com>

erikbosch commented on someip2val/cert/getCerts.sh:
```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```
No reason to include this file - better to mention https://github.com/eclipse/kuksa.val/blob/master/kuksa_certificates/genCerts.sh in a README.md?

int0x27 committed:
Removed kuksa.val keys

Signed-off-by: Ivan Kirchev <Ivan.Kirchev@bosch.com>

int0x27 committed:
Synced protofiles from kuksa.val master

- alternative solution for patching vsomeip, don't fail if already patched

Signed-off-by: Ivan Kirchev <Ivan.Kirchev@bosch.com>

int0x27 committed:
Fixed review issues

- Fixed concurrent access to ClientContext (valgrind issues on shutdown)
- minor logging fixes

Signed-off-by: Ivan Kirchev <Ivan.Kirchev@bosch.com>

erikbosch commented:
Can this one be closed @int0x27  - solved by https://github.com/eclipse/kuksa.val.feeders/pull/101?
